README.md

```diff
@@ -1 +1,149 @@
-# openlxp-xia
\ No newline at end of file
+
+# OPENLXP-XIA
+# What is it?
+OpenLXP XIA is a Python package that provides the validation functionality of extracted metadata from the source. In addition, it helps transform metadata into target metadata and further load it into indexing services. The OpenLXP XIA package does not include the metadata extraction process because extraction methods can be different for different sources. But once metadata gets extracted from the source, OpenLXP-XIA continues the V-T-V-L  (Validate-Transform-Validate-Load) cycle. 
+
+The schema files used for validation can be placed on the schema server. Currently, OpenLXP-XIA uses AWS S3 buckets as a schema server. 
+
+Below are the workflow which are performed by the OpenLXP-XIA after package installation.
+
+
+# Workflows
+The OpenLXP-XIA implements five core workflows after extracting metadata from the Specifiec source, as follows:
+
+1. `Validate`: Compares extracted learning experience metadata against the configured source metadata reference schema stored in the Experience Schema Service (XSS).
+
+2. `Transform`: Transforms extracted+validated source learning experience metadata to the configured target schema using the "XSR-to-Target" transformation map stored in the Experience Schema Service (XSS)
+
+3. `Validate`: Compares transformed learning experience metadata against the configured target metadata reference schema stored in the Experience Schema Service (XSS).
+
+4. `Load`: Pushes transformed and validated learning experience metadata to the target Experience Index Service (XIS) for further processing.
+
+5. `Log`: Records error, warning, informational, and debug events which can be reviewed and monitored.
+
+# Prerequisites
+`Python >=3.7` : Download and install python from here [Python](https://www.python.org/downloads/).
+
+
+## Environment Variables
```

Comment:
@JDTobin @SHGODE  Not sure if env values are relevant in a package, technically we do require these values so it makes sense adding it here

Reply:
I think it would make sense for any vars that are introduced by the package, but general env vars from the other openlxp services don't need to be included.

Reply:
make sense!